docs/alternator/compatibility.md

```diff
@@ -16,7 +16,21 @@ provision that - Scylla requires you to provision your cluster. You need
 to reason about the number and size of your nodes - not the throughput.
 
 When creating a table, the BillingMode and ProvisionedThroughput options
-are ignored by Scylla.
+are ignored by Scylla. Tables default to a Billing mode of `PAY_PER_REQUEST`
+and `DescribeTable` API calls will return a value of 0 for the RCUs, WCUs
+and NumberOfDecreasesToday within the response.
+
+## Scan ordering
+
+In DynamoDB, the Hash key (or partition key) determines where the item will
+be stored within DynamoDB's internal storage. Another notable difference between
+DynamoDB and Scylla comes down to the underlying hashing algorithm.
+While DynamoDB uses a proprietary hashing function, ScyllaDB implements the well-known
+[Murmur3](https://docs.scylladb.com/stable/glossary.html#term-Partitioner) algorithm.
```

Comment:
I think there's a parallel patch to add a separate "mumur3" entry to the glossary.
By the way, unfortunately Cassandra failed to implement the "well-known" murmur3 algorithm (it had a bug in the sign bit...), and Scylla (deliberately) copied its buggy implementation. But I guess this is beyond what needs to appear in this page.

Reply:
https://github.com/scylladb/scylladb/pull/11369
but not merge yet, and not a must here